nntrainer/tensor/manager.cpp

```diff
@@ -120,13 +120,16 @@ MMapedMemory::~MMapedMemory() {
   ml_logd("[MMapedMemory] buf released");
 }
 
-Manager::Manager(bool enable_gradient_memory_opt_, bool use_shared_memory_) :
+Manager::Manager(bool enable_gradient_memory_opt_,
```

Comment:
please check unittest_nntrainer_layers which initialized managers with arguments "true, false".
It should change as per this?

Reply:
We've made an offline discussion about this.
I guess we could leave this for now and later we can test with various combination of optimization :)